### PR TITLE
Added task description for zero-shot-image-classification

### PR DESCRIPTION
**Pull request by 02shanks**

### CODE REVIEW

**Comment by 02shanks** [comment]
@merveenoyan can you review the content !!

**Comment by merveenoyan** [comment]
@02shanks in this PR you added pages for `zero-shot-classification`, which is different than `zero-shot-image-classification`. `zero-shot-classification` is an NLP task (though I must admit it's not very intuitive). How would you like to proceed? Close this PR or modify it for `zero-shot-image-classification`? Note that there's also this PR: https://github.com/huggingface/hub-docs/pull/380

**Comment by 02shanks** [comment]
@merveenoyan I added a task description and deleted overlapping files. Can you review it!!

**Comment by merveenoyan** [comment]
@02shanks we've merged the #380 as I said above, the contributor wrote a definition in `data.ts` already, what I'd suggest is that you could pull latest changes for both `about.md` and `data.ts` and add your definition as the first paragraph in `about.md` as it fits there well 🙂 (under `## About the Task` section) and we'll merge it.

**Comment by 02shanks** [comment]
@merveenoyan can you review the changes !!

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot!

**Comment by 02shanks** [comment]
@merveenoyan I added the merged description below my description. Can you review the changes!!

**Comment by 02shanks** [comment]
@merveenoyan are changes made right?

File: tasks/src/zero-shot-image-classification/about.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ## About the Task
 
+Zero-shot image classification is a task in computer vision where the aim is to classify images into one of several pre-defined categories, without any prior training or knowledge of the categories.
```
Comment (merveenoyan) [inline tasks/src/zero-shot-image-classification/about.md:3]:
```suggestion
Zero-shot image classification is a computer vision task to classify images into one of several classes, without any prior training or knowledge of the classes. 
```

**Review by merveenoyan — APPROVED** [review]
I think we can merge this, I just simplified the language 🙂 Congratulations!

File: tasks/src/zero-shot-image-classification/about.md (hunk 1 above)
Comment (02shanks) [reply tasks/src/zero-shot-image-classification/about.md:3]: @merveenoyan Thank you for your much needed guidance. It really helped me a lot as it was my first contribution slightly_smiling_face

**Comment by 02shanks** [comment]
@merveenoyan thank you for guiding me patiently as this was my contribution. I learned a lot because of you!
Thank you once again!!

**Comment by merveenoyan** [comment]
@02shanks can you give me your HF username so I can add your name to the page? 🙂

**Pull request merged**